features/features.go

```diff
@@ -36,6 +36,13 @@ func AllFeatureSets() map[ClusterProfileName]map[configv1.FeatureSet]*FeatureGat
 var (
 	allFeatureGates = map[ClusterProfileName]map[configv1.FeatureSet]*FeatureGateEnabledDisabled{}
 
+	FeatureGateConsolePluginCSP = newFeatureGate("ConsolePluginCSP").
+					reportProblemsToJiraComponent("Management Console").
+					contactPerson("jhadvig").
+					productScope(ocpSpecific).
+					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
Why add a gate if you are adding it to all sets at once? I think you want to not have it as Default for now

Reply:
@JoelSpeed hmm but will the filed in this case be available by default? Cause thats what we need.

Reply:
To complete the loop from Slack, it should be introduced as TP and then promoted to Default only once QE are happy that the feature is complete and stable